src/Workleap.DomainEventPropagation.Analyzers.Tests/EventDomainAttributeUsageAnalyzerTests.cs

```diff
@@ -0,0 +1,32 @@
+namespace Workleap.DomainEventPropagation.Analyzers.Tests;
+
+public class EventDomainAttributeUsageAnalyzerTests : BaseAnalyzerTest<EventDomainAttributeUsageAnalyzer>
```

Comment:
We could add the following test cases:
- Class with IDomainEvent but with another attribute
- Class without IDomainEvent but with DomainAttribute
- Class with IDomainEvent and multiple attributes including DomainAttribute
- Record with attribute